openverse_catalog/dags/providers/provider_api_scripts/inaturalist.py

```diff
@@ -237,13 +251,18 @@ def load_catalog_of_life_names(remove_api_files: bool):
                     f.write(z.read(vernacular_file))
                 logger.info(f"Extracted raw file: {OUTPUT_DIR}/{vernacular_file}")
         # set up for loading data
-        pg = PostgresHook(POSTGRES_CONN_ID)
+        pg = PostgresHook(
+            default_statement_timeout=PostgresHook.get_execution_timeout(task)
+        )
         COPY_SQL = (
             "COPY inaturalist.{} FROM STDIN "
             "DELIMITER E'\t' CSV HEADER QUOTE E'\b' NULL AS ''"
         )
         COUNT_SQL = "SELECT count(*) FROM inaturalist.{};"
         # upload vernacular names file to postgres
+        pg.run(
+            ""
+        )  # just to set the timeout, because it's not automatic for copy_expert
```

Comment:
Maybe instead, we could have a separate function for cases like this that more explicitly runs the statement timeout? Like `pg.run_statement_timeout()`?

Reply:
That's better. I was considering when and whether I should get into the guts of more of the hook functions (like running a list of SQLs, which I didn't see in our code base or running copy_expert), but that's a nice solution to avoid the issue with copy expert at least.